supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -27,6 +28,13 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
 
   private implicit val ec = context.dispatcher
   private val maxRetries = 10
+  // FIXME: Determined empirically that we start to get errors when the batch request approaches 15MB.
+  // Find out what the real value is
+  private val maxBatchRequestSize = 14 * 1024 * 1024
+  private val requestTooLargeException = new JesApiException(new IllegalArgumentException(
+    "The JES creation request exceeds the maximum size allowed. " +
+      "If you have a task with a very large number of inputs and / or outputs in your workflow you should try to reduce it."
```

Comment:
@geoffjentry If they truly have a task that generates a request > 15 MB

1. They have a ridiculous amount of inputs (that would probably take forever to JES to localize anyway).
As a benchmark they would need approximately 7 million input files looking like 'gs://cloud-cromwell-dev/mock/mock_firecloud_workspace_folder/mock_firecloud_submission_folder/mock_cromwell_workflow_name/mock_workflow_uuid/mock-call-alias-directory/file1.txt' to go over the limit.

I would say that there must be a better solution to what they're trying to do.

2. If we still want to try something we could try to submit it to JES directly instead of batching it but even if not the same they must have a limit too.